api/HTMLScriptElement.json

```diff
@@ -206,16 +206,9 @@
             "firefox_android": {
               "version_added": "4"
             },
-            "ie": [
-              {
-                "version_added": "10"
-              },
-              {
-                "version_added": "4",
-                "version_removed": "10",
-                "notes": "Non-standard implementation"
```

Comment:
Looking at this again now as I was about to merge with the "useless and inaccurate" note in the commit message, I note that this actually lines up with a comment in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#Browser_compatibility saying "In versions prior to Internet Explorer 10, it implemented defer by a proprietary specification. Since version 10 it conforms to the W3C specification." Neither are great notes, but I think it'd be best to make them consistent rather than removing this note. Something was different about `defer` before IE10, apparently.

I'll back out this change to get the rest landed.

Reply:
I've filed https://github.com/mdn/browser-compat-data/issues/7258 to follow up on that.